lms/static/sass/course/courseware/_sidebar.scss

```diff
@@ -1,234 +1,168 @@
 .course-index {
-  @extend .sidebar;
-  @extend .tran;
-  @include border-right(1px solid $border-color-2);
-  @include border-radius(3px, 0, 0, 3px);
-
-  #open_close_accordion {
-    display: none;
-  }
-
-  header {
-    max-height: 47px;
-
-    h2 {
-      white-space: nowrap;
-    }
-  }
-
-  div#accordion {
-    width: auto;
-    font-size: 14px;
+    @extend .sidebar;
```

Comment:
NIT: I know you're just moving existing code, but this extend is gross and causes extra CSS bloat.